index.html

```diff
@@ -1,12 +1,368 @@
+                <div class="camp">
+                    <h5>
+                        CANOPY
+                    </h5>
+                </div>
+                <div class="year">
+                    <ul class="list flex">
+                        <li>Back And Dev</li>
+                        <li>2015</li>
+                    </ul>
+                </div>
+            </div>
+            <div class="task">
+                <p>
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+            </div>
+            <div class="web flex">
+                <div class="right flex">
+                    <ul>
+                        <li>
+                            <a href="#">HTML</a>
+                        </li>
+                        <li>
+                            <a href="#">CSS</a>
+                        </li>
+                        <li>
+                            <a href="#">JavaScript</a>
+                        </li>
+                    </ul>
+                </div>
+                <div class="left"></div>
+            </div>
+            <div class="link flex">
+                <a href="#">
+                    See project
+                </a>
+            </div>
+        </div>
+
+        <!-- End of First card -->
+
+        <!-- Start Second card -->
+
+
+        <div class="middle_section">
+            <div class="pic">
+                <img src="Media/Snapshoot Portfolio1.png" alt="Snapshoot pic">
+            </div>
+            <div class="name">
+                <h1>
+                    Multi-Post Stories
+                </h1>
+            </div>
+            <div class="descript flex">
+                <div class="camp">
+                    <h5>
+                        CANOPY
+                    </h5>
+                </div>
+                <div class="year">
+                    <ul class="list flex">
+                        <li>Back And Dev</li>
+                        <li>2015</li>
+                    </ul>
+                </div>
+            </div>
+            <div class="task">
+                <p>
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+            </div>
+            <div class="web">
+                <div class="right">
+                    <ul>
+                        <li>
+                            <a href="#">HTML</a>
+                        </li>
+                        <li>
+                            <a href="#">CSS</a>
+                        </li>
+                        <li>
+                            <a href="#">JavaScript</a>
+                        </li>
+                    </ul>
+                </div>
+                <div class="left"></div>
+            </div>
+            <div class="link flex">
+                <a href="#">
+                    See project
+                </a>
+            </div>
```

Comment:
- [ ] Kindly consider applying the button `pressed` styling interaction by using `:active` pseudoclass following the figma color guidelines. you may implement this in all the buttons.
![as](https://user-images.githubusercontent.com/96683843/235983651-89d4caf9-d366-4ae6-bf04-57650d5b3bd5.jpg)